Doc/library/test.rst

```diff
@@ -324,7 +324,7 @@ The :mod:`test.support` module defines the following constants:
 
 .. data:: Py_DEBUG
 
-   True if Python is built with the :c:macro:`Py_DEBUG` macro defined: if
+   True if Python is built with the :c:macro:`!Py_DEBUG` macro defined: if
```

Comment:
It looks like we refer to this macro from quite a few places in the docs:

```
c-api/init_config.rst:      Needs a :ref:`debug build of Python <debug-build>` (the ``Py_DEBUG`` macro
c-api/init_config.rst:      Needs a :ref:`debug build of Python <debug-build>` (the ``Py_DEBUG`` macro
c-api/intro.rst:   If Python is :ref:`built in debug mode <debug-build>` (if the ``Py_DEBUG``
c-api/intro.rst:Compiling the interpreter with the :c:macro:`Py_DEBUG` macro defined produces
c-api/intro.rst::c:macro:`Py_DEBUG` is enabled in the Unix build by adding
c-api/intro.rst:not-Python-specific :c:macro:`_DEBUG` macro.  When :c:macro:`Py_DEBUG` is enabled
library/test.rst:.. data:: Py_DEBUG
library/test.rst:   True if Python is built with the :c:macro:`Py_DEBUG` macro defined: if
using/configure.rst:* Define ``Py_DEBUG`` and ``Py_REF_DEBUG`` macros.
using/configure.rst:* Add runtime checks: code surrounded by ``#ifdef Py_DEBUG`` and ``#endif``.
using/configure.rst:   ``Py_DEBUG`` macro no longer implies the ``Py_TRACE_REFS`` macro (see the
using/configure.rst:   :ref:`Build Python in debug mode <debug-build>`: define the ``Py_DEBUG``
whatsnew/3.10.rst:  <debug-build>` (if the ``Py_DEBUG`` macro is defined). In the limited C API,
whatsnew/3.8.rst:``Py_DEBUG`` macro no longer implies the ``Py_TRACE_REFS`` macro, which
```

Maybe a better way of solving this one would be to add documentation for it, that we can link to from all these places?

Reply:
`Doc/using/configure.rst` seems to be the best place for this. In fact, there is this:

```
.. _debug-build:

Python Debug Build
------------------

A debug build is Python built with the :option:`--with-pydebug` configure
option.

Effects of a debug build:

...

Debug options
-------------

.. option:: --with-pydebug

   :ref:`Build Python in debug mode <debug-build>`: define the ``Py_DEBUG``
   macro (disabled by default).
```

Given the fact that `Py_DEBUG` is just the marker of a debug build, I'd suggest linking to the `_debug-build` anchor.